README.md

```diff
@@ -39,6 +39,10 @@ If you wish you can run this command under a virtual environment.
    when getting images, Avocado will create a snapshot to avoid conflicts. Also
    domains created by tests have unique names to avoid collisions.
 
+ * Creating domains easily: You can use the `.create_domain()` method to create a
+   generic domain, based on a template (Jinja2).  `Domain.from_xml_path()` is
+   also available, if you need to use a more specific XML file.
```

Comment:
Let's swap the clauses: "If you want to use an already existing XML domain description, no problem, use `Domain.from_xml_path()` instead."